Formula/mk-configure.rb

```diff
@@ -0,0 +1,22 @@
+class MkConfigure < Formula
+  desc "Lightweight replacement for GNU autotools"
+  homepage "https://github.com/cheusov/mk-configure"
+  url "https://downloads.sourceforge.net/project/mk-configure/mk-configure/mk-configure-0.29.3/mk-configure-0.29.3.tar.gz"
+  sha256 "1f483f717a09d06c82a6f3f284dd34234f54ead032e6987d33f3cf8109490c05"
+
+  depends_on "bmake"
+  depends_on "makedepend"
+
+  def install
+    ENV["PREFIX"] = prefix.to_s
+    ENV["MANDIR"] = man.to_s
+
+    system "bmake", "all"
+    system "bmake", "install"
+    (share/"doc/mk-configure").install("presentation/presentation.pdf")
```

Comment:
This is just `doc.install` in long-form, no?

Reply:
You are right. Thanks for pointing out!